### PR TITLE
Gzweb support

### PR DESCRIPTION
**Pull request by kapilPython**
*Issue #, if available:*
Adding Gzweb Support

*Description of changes:*
1. change of all model.sdf files to make it compatible to gzweb, the changes still work with gazebo and ros.
2. Added changes to README to allow use of the models with Gzweb.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by cheng-kevin** [comment]
I've tested this in the [Cloudwatch Sample App](https://github.com/aws-robotics/aws-robomaker-sample-application-cloudwatch) in ROS Melodic, Gazebo 9.14 on Ubuntu 18.04 and it seems to be working.

**Comment by cheng-kevin** [comment]
Also tested in Gzweb 1.4.0,  in ROS Melodic, Gazebo 9.14 on Ubuntu 18.04 and it seems to be working. Node 8.11.3, nodejs 12.19.0, npm 5.6.0. 

Gzweb installation was troublesome.  Needs the exact node versions listed above.  Node-gyp was installed globally

File: README.md [hunk 1]
```diff
@@ -25,6 +29,34 @@ export GAZEBO_MODEL_PATH=`pwd`/models
 gazebo worlds/small_house.world
 ```
 
+## Example: Running this world on Gazebo headless and running the UI on Gzweb
+*Tested in ROS Kinetic, Gazebo 7 with node version 10.22.1*
```
Comment (cheng-kevin) [inline README.md:33]: Could you include a more dire warning regarding the potential pain of installing Gzweb, outside of these tested versions?  ( I tested in ROS Melodic, Gazebo 9.14 , Node 8.11.3)

**Review by mm318 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (kapilPython) [reply README.md:33]: For avoiding the pain of installation for gzweb I created a dockerfile which works with ROS kinetic, gazebo 7.16 and node version 10.22.1 the image could be easily pulled from https://hub.docker.com/r/kapildesh/gzweb7.

Comment (cheng-kevin) [reply README.md:33]: Great, thank you!

**Review by cheng-kevin — APPROVED** [review]
Looks good to me

**Pull request merged**